packages/web3-eth-accounts/src/rlp/index.ts

```diff
@@ -0,0 +1,204 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// @ethereumjs/rlp version 4.0.1
+import { NestedUint8Array } from '../common/types';
+import { bytesToHex, concatBytes, encodeLength, Input, parseHexByte, toBytes } from './utils';
+
+export interface Decoded {
+	data: Uint8Array | NestedUint8Array;
+	remainder: Uint8Array;
+}
+
+/**
+ * RLP Encoding based on https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/
+ * This function takes in data, converts it to Uint8Array if not,
+ * and adds a length for recursion.
+ * @param input Will be converted to Uint8Array
+ * @returns Uint8Array of encoded data
+ * */
+const encode = (input: Input): Uint8Array => {
```

Comment:
These files should be removed and eth/rlp lib dependency should be used back. Suggested by @mpetrunic